### PR TITLE
[SMALLFIX]Add javadocs for setupMasterCluster

### PR DESCRIPTION
**Pull request by summerDG**
Add javadocs for JournalShutdownIntegrationTest#setupMultiMasterCluster and JournalShutdownIntegrationTest#setupSingleMasterCluster

### CODE REVIEW

File: tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java [hunk 1]
```diff
@@ -148,6 +148,9 @@ private void reproduceAndCheckState(int successFiles) throws Exception {
     registry.stop();
   }
 
+  /**
+   * Setup and start multiple master cluster.
```
Comment (apc999) [inline tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java]: NIT: Setup -> Sets up

File: tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java [hunk 2]
```diff
@@ -157,6 +160,9 @@ private MultiMasterLocalAlluxioCluster setupMultiMasterCluster() throws Exceptio
     return cluster;
   }
 
+  /**
+   * Setup and start single master cluster.
```
Comment (apc999) [inline tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java]: ditto

File: tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java [hunk 3]
```diff
@@ -148,6 +148,9 @@ private void reproduceAndCheckState(int successFiles) throws Exception {
     registry.stop();
   }
 
+  /**
+   * Sets up and start multiple master cluster.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java]: Sets up and starts a cluster containing multiple masters.

File: tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java [hunk 4]
```diff
@@ -157,6 +160,9 @@ private MultiMasterLocalAlluxioCluster setupMultiMasterCluster() throws Exceptio
     return cluster;
   }
 
+  /**
+   * Sets up and start single master cluster.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java]: Sets up and starts a cluster containing a single master.

File: tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java [hunk 5]
```diff
@@ -148,6 +148,9 @@ private void reproduceAndCheckState(int successFiles) throws Exception {
     registry.stop();
   }
 
+  /**
+   * Sets up and starts multiple master cluster.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java]: Sets up and starts a multi-master cluster.

File: tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java [hunk 6]
```diff
@@ -157,6 +160,9 @@ private MultiMasterLocalAlluxioCluster setupMultiMasterCluster() throws Exceptio
     return cluster;
   }
 
+  /**
+   * Sets up and starts single master cluster.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java]: Sets up and starts a single master cluster.

**Pull request merged**